### PR TITLE
Update Indonesia Translation

### PR DESCRIPTION
**Pull request by kuchingneko28**
Corrected several words

### CODE REVIEW

**Comment by robifr** [comment]
i guess it's better for us to get into discussion.

File: assets/translations/id.json [hunk 1]
```diff
@@ -141,223 +141,223 @@
...
-    "useVersionCodeAsOSVersion": "Gunakan versi kode aplikasi sebagai versi yang terdeteksi OS",
+    "useVersionCodeAsOSVersion": "Gunakan versionCode aplikasi sebagai versi yang terdeteksi oleh OS",
     "requestHeader": "Header permintaan",
     "useLatestAssetDateAsReleaseDate": "Gunakan unggahan aset terbaru sebagai tanggal rilis",
-    "defaultPseudoVersioningMethod": "Metode versi semu standar",
-    "partialAPKHash": "Hash APK parsial",
-    "APKLinkHash": "Hash tautan APK",
-    "directAPKLink": "Tautan APK langsung",
-    "pseudoVersionInUse": "Versi semu sedang digunakan",
+    "defaultPseudoVersioningMethod": "Metode Penomoran Versi Semu",
```
Comment (robifr) [inline assets/translations/id.json:302]: you forgot the "standard" part. albeit i was thinking about "metode standar dalam penomoran versi semu" is actually better, but it's mouthful. so keep as is i guess.

File: assets/translations/id.json [hunk 2]
```diff
@@ -141,223 +141,223 @@
...
-        "one": "Menghapus {n} catatan (sebelum = {before}, setelah = {after})",
-        "other": "Menghapus {n} catatan (sebelum = {before}, setelah = {after})"
+        "one": "Menghapus {n} log (sebelum = {before}, setelah = {after})",
+        "other": "Menghapus {n} log (sebelum = {before}, setelah = {after})"
     },
     "xAndNMoreUpdatesAvailable": {
-        "one": "{} dan 1 aplikasi lainnya mendapat pembaruan.",
-        "other": "{} dan {} aplikasi lainnya mendapat pembaruan."
+        "one": "{} dan 1 aplikasi lainnya memiliki pembaruan.",
+        "other": "{} dan {} aplikasi lainnya memiliki pembaruan."
```
Comment (robifr) [inline assets/translations/id.json:360]: agree for this one, because it's either "mendapatkan" or "memiliki".

**Review by robifr — CHANGES_REQUESTED** [review]
i can't comment for every lines as most of the reason are same. however, the previous translation aims to sounds more natural to our user. most of the translation are misleading too. 

there are some stuff to do after reviewing this PR. 
- improve "rate limit" term by checking for how other app/project use the term to make it consistent with others.
- check whether "input" is a proper term in indonesian KBBI or not, to improve line 95.

**Comment by ImranR98** [comment]
Will wait for their response.

**Comment by kuchingneko28** [comment]
> i can't comment for every lines as most of the reason are same. however, the previous translation aims to sounds more natural to our user. most of the translation are misleading too.
> 
> there are some stuff to do after reviewing this PR.
> 
> * improve "rate limit" term by checking for how other app/project use the term to make it consistent with others.
> * check whether "input" is a proper term in indonesian KBBI or not, to improve line 95.

I was trying to keep it consistent with the words used, simplifying it while preserving the context. If you find any text that feels incorrect, feel free to change it

**Pull request closed without merging**